brainstormforce/custom-fonts
# Search functionality related QA fixes for Custom Font Plugin.

### Description
- Adjusted Search Route in default route itself to avoid code duplication.
- Fixed issue which limited fonts to only 10.
- Added a clear button to search input.

### Screenshots
https://d.pr/i/FlZ3Al

### Types of changes
Improvements

### How has this been tested?
- Add more than 10 fonts. Check if all are visible.
- Try searching and clearing input query.

### Checklist:
